back/spec/services/json_forms_service_spec.rb

```diff
@@ -211,6 +212,26 @@
               description: 'Which councils are you attending in our city?'
             },
             scope: '#/properties/field7'
+          },
+          {
+            type: 'Control',
+            scope: '#/properties/field8',
+            label: 'Please indicate how strong you agree or disagree with the following statements.',
+            options: {
+              description: 'Which councils are you attending in our city?',
+              input_type: 'matrix_linear_scale',
+              statements: [
+                { key: 'send_more_animals_to_space', label: 'We should send more animals into space' },
+                { key: 'ride_bicycles_more_often', label: 'We should ride our bicycles more often' }
+              ],
+              linear_scale_label1: 'Strongly disagree',
+              linear_scale_label2: '',
+              linear_scale_label3: '',
+              linear_scale_label4: '',
+              linear_scale_label5: 'Strongly agree',
+              linear_scale_label6: '',
+              linear_scale_label7: ''
```

Comment:
Mentioned it to Edwin the other day when he add up to 11 for linear scale, but think a good future refactor would be to just have one `linear_scale_labels` column on custom fields with all the labels in there as one json blob

Reply:
I agree it would be an improvement. Either a JSON blob or even a separate table/relationship.